Assignment1/generate_datapoints.py

```diff
@@ -0,0 +1,66 @@
+# int numpoints
+# list points
+# double length1
+# double radius1
+# double epsilon
+# double area1
+# int Q_PTS
+# int quads
+# double dist
+# int N_PTS
+# int K_PTS
+# int A_PTS
+# int B_PTS
+# int C_PTS
+# int D_PTS
+# int E_PTS
+# int F_PTS
+# int G_PTS
+# double length2
+# double radius2
+# double area2
+# list LCM
+# list PUV
+
+
+import random
+import pickle
+
+
+def generate_datatpoints(x):
```

Comment:
Please include documentation for each function (short comment explaining what it does, if you're using VScode I recommend the extension "Python Docstring Generator", I just downloaded it and it's really really easy)